crates/host/src/wasmbus/mod.rs

```diff
@@ -895,6 +895,38 @@ async fn create_lattice_metadata_bucket(
 impl Host {
     const HEARTBEAT_INTERVAL: Duration = Duration::from_secs(30);
 
+    const NAME_ADJECTIVES: &str = "
+    autumn hidden bitter misty silent empty dry dark summer
+    icy delicate quiet white cool spring winter patient
+    twilight dawn crimson wispy weathered blue billowing
+    broken cold damp falling frosty green long late lingering
+    bold little morning muddy old red rough still small
+    sparkling bouncing shy wandering withered wild black
+    young holy solitary fragrant aged snowy proud floral
+    restless divine polished ancient purple lively nameless
+    gray orange mauve
```

Comment:
I didn't convert it to `&[&str]`, string look better maintained.
```rust
const NAME_ADJECTIVES: &[&str] = [
"autumn",
"hidden",
"bitter",
... // A very long list
]

Reply:
I would prefer an array here, but I understand the reasoning and we can always optimize later if we need to (we most probably will not)

Reply:
I tried the array and it is very long after formatting.:joy: